src/Nancy/ModelBinding/DefaultModelBinderLocator.cs

```diff
@@ -27,7 +27,11 @@ public class DefaultModelBinderLocator : IModelBinderLocator
         public DefaultModelBinderLocator(IEnumerable<IModelBinder> binders, IBinder fallbackBinder)
         {
             this.fallbackBinder = fallbackBinder;
-            this.binders = binders;
+
+            if (binders != null)
+            {
+                this.binders = binders.ToArray();

```

Comment:
We can't wrap it because it's `IEnumerable<T>` which means we need to materialize the collection first using either `ToList` or `ToArray`.